web-wallet/src/routes/(welcome)/setup/+page.svelte

```diff
@@ -41,6 +41,10 @@
 	</div>
 </section>
 
+<footer>
+	<span>Web Wallet v{import.meta.env.APP_VERSION} ({import.meta.env.APP_BUILD_INFO})</span>
```

Comment:
It make sense to add this to the settings page as well, as this info is only displayed on the setup page and nowhere else.

Reply:
> It make sense to add this to the settings page as well, as this info is only displayed on the setup page and nowhere else.

Yes, indeed it'll be added even there.